### ISSUE TITLE
MacOS Desktop Client 4.1 freezes immediately after startup 

### ISSUE THREAD

**Issue by bendschs** [nextcloud/desktop#4187]
MacOS Desktop Client 4.1 freezes immediately after startup.

This happens on Catalina and Monterey. My Nextcloud Server is running 23.0.
I did not find anything useful from the serverlogs, the client is showing me a couple of files that could not be synced because they have "white space at the end of filename" (just before freezing), which is not even true, i checked all the files the client mentioned none had actually a white space at the end.

![Bildschirmfoto 2022-01-20 um 15 16 10](https://user-images.githubusercontent.com/17085451/150356432-ac7cdadf-ff30-46ac-9aa7-3b841bcf2456.png)

On server side i get a very unspecific  "sabre/dav/Exception: Unkonw error while seeking content"

![Bildschirmfoto 2022-01-20 um 15 26 12](https://user-images.githubusercontent.com/17085451/150357631-f51e759d-2439-4ecd-8fdd-11038a849aa2.png)

**Comment by asquiart** [comment]
Same issue here on Linux.
Server logs same as above.
As soon as it starts syncing it freezes and pins one CPU to 100%.

https://pastebin.com/N8F3kxns

**Comment by bendschs** [comment]
just to spice it up a little bit: 

![Bildschirmfoto 2022-01-21 um 17 33 37](https://user-images.githubusercontent.com/17085451/150564610-03d6466c-5bb2-46de-a4a0-5ff8bddf6c35.png)

maybe the dropbox m1-release actually works 👿

**Comment by d-3-s** [comment]
I observed that bug as well. I did a bit more debugging and can report the following.

I am almost sure that it is related to the client and has nothing todo with the server. I had my client connected to a server of 21.0.1 server and a 23.0.0. server. 
I have certain files, which let the server crash as soon as I try to place it in one of the folders.
The way I debugged it is was that I killed nextcloud client, removed half of the files, and checked if it crashed or not, and then added them step by step till I found the file.

I have files which look totally normal and have already been on nextcloud, and lead to the crash once I moved them.

I managed to reproduce now a case which leads on my client always to a crash.

1. I create a folder .git and wait till it is synced
2. I upload the attached rtf file (not the zipped version) [test.rtf.zip](https://github.com/nextcloud/desktop/files/7936015/test.rtf.zip)
Edit: It seams a lot of files which I create inside ".$foldername" lead to the crash. touch test.txt inside a folder ".test" is o.k. but as soon as I edit it and save it with asdf it crahes 
3. I wait a few seconds and the client will crash

Resolving it:
1. kill the client
2. remove this file --> everything is fine

I originally also tried to manually delete the local caches 

- /Users/$User/Library/Caches/Nextcloud
- the 3 files in the root folder of the nextcloud folder (*.db, *.db-shm, *.db-wal)

To me it looked like it is not needed or helping anything to delete them.
